tools/build_variables.bzl

```diff
@@ -155,7 +155,7 @@ core_sources_full_mobile = [
     "torch/csrc/jit/frontend/convert_to_ssa.cpp",
     "torch/csrc/jit/frontend/exit_transforms.cpp",
     "torch/csrc/jit/frontend/inline_loop_condition.cpp",
-    "torch/csrc/jit/frontend/ir_emitter.cpp",
+    "torch/csrc/jit/api/compilation_unit.cpp",
```

Comment:
While we're moving code around, what are people's opinions on keeping `CompilationUnit` in the `api` folder? It is part of the public API, but not to the same extent as `Module`, `Object`, and even `Method`.